.dependabot/config.yml

```diff
@@ -1355,4 +1355,17 @@ update_configs:
     automerged_updates:
     - match:
         dependency_name: demisto/*
-        update_type: all
\ No newline at end of file
+        update_type: all  - package_manager: python
+    directory: /docker/pcap-miner
+    update_schedule: live
+    automerged_updates:
+    - match:
+        update_type: semver:minor
+  - package_manager: docker
+    directory: /docker/pcap-miner
+    update_schedule: daily
+    automerged_updates:
+    - match:
+        dependency_name: demisto/*
+        update_type: all
+
```

Comment:
@yaakovi is this replacement ok? I think it was automatically generated by the docker script.